pages/about_new.tsx

```diff
@@ -4,109 +4,150 @@ import {
+    >
       <AboutUsHeroSection theme={theme} extraSmallScreen={extraSmallScreen} />
-      <WhatWeDoSection theme={theme} extraSmallScreen={extraSmallScreen} />
-      <OurValueSection theme={theme} extraSmallScreen={extraSmallScreen} />
-      <OurVisionSection theme={theme} extraSmallScreen={extraSmallScreen} />
+      <WhatWeDoSection theme={theme} />
+      <OurValueSection theme={theme} />
+      <OurVisionSection theme={theme} />
       <OurTeamSection theme={theme} extraSmallScreen={extraSmallScreen} />
-    </div>
+    </Container>
   )
 }
 
 export default withBasicLayout(AboutPage)
 
-const AboutUsHeroSection = ({ theme, extraSmallScreen }) => (
-  <Stack
-    sx={{
-      backgroundColor: theme.palette.primary.main,
-      padding: extraSmallScreen ? '1rem': '2rem',
-    }}
-  >
-    <Grid container direction="row" justifyContent="center" alignItems="center">
-      <Grid item md={4} xs={12}>
-        <Box sx={{ gap: '2.5rem', display: 'grid', margin: '2rem' }}>
-          <Typography
-            variant={extraSmallScreen ? "displayMedium" : "displayLarge"}
-            sx={{ color: theme.palette.primary.contrastText }}
-          >
-            About us
-          </Typography>
-          <Typography
-            variant="bodyLarge"
-            sx={{ color: theme.palette.primary.contrastText }}
-          >
-            Open Seattle comprises a group of seasoned professionals with
-            experience in web development, software engineering, digital
-            strategy, visual and content design, and more. We leverage this vast
-            experience to transform the nonprofit sector—to make it more
-            efficient, effective, and accessible to all.
-          </Typography>
-        </Box>
-      </Grid>
-      <Grid item md={4} xs={12}>
-        <img src={AboutUsImage.src} alt="About Us page graphic"></img>
-      </Grid>
-    </Grid>
-  </Stack>
+const AboutUsSection = ({ backgroundColor, children }) => (
+  <SectionContainer backgroundColor={backgroundColor}>
+    <Stack
+      gap={{ xs: '64px', md: '80px' }}
+      sx={{
+        textAlign: 'center',
+      }}
+      maxWidth={'880px'}
+    >
+      {children}
+    </Stack>
+  </SectionContainer>
 )
 
-const WhatWeDoSection = ({ theme, extraSmallScreen }) => (
-  <SectionContainer
-    sx={{
-      backgroundColor: designColor.white,
-      padding: extraSmallScreen ? '1rem' : '2rem 12rem 2rem 12rem',
-    }}
-    maxWidth={false}
-  >
-    <Box sx={{ gap: '2rem', display: 'grid', margin: '2rem' }}>
-      <Typography variant="headlineMedium">What we do</Typography>
-      <Typography variant="bodyLarge">
-        Open Seattle partners with other nonprofits in order to amplifying their
-        impact, and to uplift communities with technology.
-      </Typography>
+const AboutUsHeroSection = ({ theme, extraSmallScreen }) => (
+  <SectionContainer backgroundColor={theme.palette.primary.main}>
```

Comment:
Minor thing but from the figma tablet design, it looks like the hero width should match the width of the sections below it. 
I also wonder if making this hero into a component would be useful for keeping things consistent (or if it's overkill lol)